packages/core/src/sass/app/modules/account-signup.scss

```diff
@@ -2,6 +2,27 @@
 .account-signup {
     margin: 1.6rem 1.6rem 0;
 
+    @include mobile {
+        width: calc(100% + 50px);
+        border-radius: 10px;
+        padding: 20px;
+        position: relative;
+        left: 50%;
+        transform: translate(-50%);
+        margin: 0;
+        background-color: #151717;
```

Comment:
Question: do we already have this color in the system? We usually set color as a variable and reuse it across our css